x/inflation/keeper/integration_test.go

```diff
@@ -0,0 +1,53 @@
+package keeper_test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	incentivestypes "github.com/tharsis/evmos/x/incentives/types"
+)
+
+var _ = Describe("Integration", Ordered, func() {
+
+	BeforeEach(func() {
+		s.SetupTest()
+	})
+
+	Describe("Commiting a block", func() {
+		addr := s.app.AccountKeeper.GetModuleAddress(incentivestypes.ModuleName)
+		before := s.app.BankKeeper.GetBalance(s.ctx, addr, denomMint)
+
+		Context("before an epoch ends", func() {
+			BeforeEach(func() {
+				s.CommitAfter(time.Minute)    // Start Epoch
+				s.CommitAfter(time.Hour * 23) // End Epoch
+			})
+
+			It("should not allocate funds to usage incentives", func() {
+				actual := s.app.BankKeeper.GetBalance(s.ctx, addr, denomMint)
+				Expect(actual.IsZero()).To(BeTrue())
+			})
+		})
+
+		Context("after an epoch ends", func() {
+			BeforeEach(func() {
+				s.CommitAfter(time.Minute)    // Start Epoch
+				s.CommitAfter(time.Hour * 24) // End Epoch
+			})
+
+			It("should allocate funds to usage incentives", func() {
```

Comment:
should we do one for team, community pool and fee collector (i.e validators)?

Reply:
also test:

- it should allocate funds to community pool if team address is not defined

Reply:
@fedekunze We discussed removing the team address logic, so there is no team address anymore.

Reply:
ah right, nvm. We can add one for the community pool only then